.gitignore

```diff
@@ -115,3 +115,5 @@ venv.bak/
 
 # ansible
 *.retry
+
+custom.*
```

Comment:
what is that?

Reply:
a way to pass custom docker files and envs files (should be ignored as it may contains secrets)